src/main/external-resources/UMS.conf

```diff
@@ -196,7 +196,7 @@ http_engine_v2 =
 # Prevent system idle sleep
 # -------------------------
 # If your UMS host has a power-saving mode, use this option to prevent the
-# system from hibernating or sleeping. Currently only Windows and macOS are
+# system from hibernating or sleeping. Currently only Windows and Mac OS are
 # supported. There are three modes:
```

Comment:
Not wrong. That's said i never saw "your" wrinting before, sorry to not be a Mac fan :package: 
As i guess that you prefer the other, weird to me, variant, i'll revert it.

I'll go to sleep smarter ;-)